runtime/protos/protos/receipt.proto

```diff
@@ -8,31 +8,23 @@ message CallbackInfo {
     string receiver = 3;
 }
 
-message AccountingInfo {
-    string originator = 1;
-    google.protobuf.StringValue contract_id = 2;
-}
-
 message AsyncCall {
     uint64 amount = 1;
-    uint32 mana = 2;
     bytes method_name = 3;
     bytes args = 4;
     CallbackInfo callback = 5;
-    AccountingInfo accounting_info = 6;
+    string refund_account = 7;
+
+    // Removed fields.
```

Comment:
I think at this stage we can just reorder the fields. We shouldn't have issues with versions yet.

Reply:
I would keep it on a safe side for now and then clean it up once we stabilize protos for the MainNet. We still sometimes use nearlib with nearcore of different versions. E.g. this week when we were using `0.1.4` nearcore and had to rollback storage API changes.